app/app/static/custom/ourscript.js

```diff
@@ -0,0 +1,117 @@
+var timers = [];
+var timerIntervals = [];
+
+function showAddToQueue() {
+	$('#addToQueue').css('display', 'block');
+}
+
+function hideAddToQueue() {
+	document.getElementById('addToQueueForm').reset();
+	$('#addToQueue').css('display', 'none');
+}
+
+$('#addToQueueForm').submit(function (e) {
+	e.preventDefault();
+
+	let name = document.getElementById('studentName').value;
+	let id = document.getElementById('studentNumber').value;
+	let unit = document.getElementById('unitCode').value;
+	let team = document.getElementById('team').value;
+	let enquiry = document.getElementById('enquiryType').value;
+
+	if (team == 'STUDYSmarter') {
+		var table = $('#SSQueueTable tbody');
+	} else {
+		var table = $('#libQueueTable tbody');
+	}
+
+	table.append(
+		'<tr id="' +
+			id +
+			'" class="initialTime">' +
+			'<td>' +
+			name +
+			'</td>' +
+			'<td>' +
+			id +
+			'</td>' +
+			'<td>' +
+			unit +
+			'</td>' +
+			'<td class="text-right">' +
+			enquiry +
+			'</td>' +
+			'<td class="text-right">' +
+			'<label id="minutes' +
+			id +
+			'">00</label><label id="colon">:</label><label id="seconds' +
+			id +
+			'">00</label>' +
+			'</td>' +
+			'<td class="td-actions text-right"><button type="button" rel="tooltip" class="btn btn-danger" onclick="deleteRow(this)"><i class="material-icons">close</i></button></td>' +
+			'</tr>'
+	);
+	hideAddToQueue();
+	timers[id] = 0;
+	console.log(timers[id]);
```

Comment:
is this console log still needed? or just used for debugging? might be worth removing this (if a codebase has too much of this... it becomes very hard to find which console logging is important)